airflow-core/src/airflow/migrations/versions/0094_3_1_0_remove_team_id.py

```diff
@@ -0,0 +1,148 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Drop ``id`` column from ``team`` table and make ``name`` the primary key.
+
+Revision ID: b12d4f98a91e
+Revises: 665854ef0536
+Create Date: 2025-12-05
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "b12d4f98a91e"
+down_revision = "665854ef0536"
+branch_labels = None
+depends_on = None
+airflow_version = "3.1.0"
```

Comment:
@vincbeck 3.1.0 doesn't seem right, either this should be 3.2.0, or 3.1.5 and backport it (I think the former?)

Reply:
My bad, it definitely should be 3.2.0. I'll create a PR. Thanks for the catch